core/dbt/task/compile.py

```diff
@@ -43,19 +43,32 @@ def raise_on_first_error(self):
         return True
 
     def get_node_selector(self) -> ResourceTypeSelector:
+        if getattr(self.args, "inline", None):
+            resource_types = [NodeType.SqlOperation]
+        elif getattr(self.args, "select", None):
```

Comment:
This will be a behavior change, we can do it in task_end_message if that change is not acceptable.(I don't know whether it is or not)